src/Umbraco.Web.UI.Client/src/common/services/javascriptlibraryhelper.service.js

```diff
@@ -1,19 +1,19 @@
 ﻿(function () {
     'use strict';
 
-    function momenthelperService($q, $http, umbRequestHelper) {
+    function javascriptLibraryHelperService($q, $http, umbRequestHelper) {
```

Comment:
@Shazwazza @hartvig would it be better to call this either `javascriptLibraryHelper` or `javascriptLibraryService`?

Reply:
@bjarnef yup I would agree, it's really either a helper or a service, I'd go with service :) //cc @hartvig